### PR TITLE
Updated README.

### PR DESCRIPTION
**Pull request by Samarth-Agarwal1**
Updated the README to include instructions on building, installing and running. Deleted a lot of info which seems to be outdated now that a lot of refactoring has been done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,180 +4,54 @@
...
-2. Setup virtualenv and install dependencies
+### Installing
 
+Run the following command:
 ```
-cd ID3_project_sickkids
-virtualenv id3_env
-source id3_env/bin/activate
-pip3 install -r requirements.txt
+pip install dist/id3_variants_training-0.0.0-py3-none-any.whl
```
Comment (ljdursi) [inline README.md]: I'm not against this but is there a reason to prefer this two step approach (setup sdist, pip install dict) to `pip install .` ?

File: README.md [hunk 2]
```diff
@@ -4,180 +4,54 @@
...
-### Installing and starting candig_server
-
-Additionally, you can host the candig_server locally if you want to test the capabilities
+positional arguments:
+  config_path  path to the config file that contains vcf file paths
+  model_file   path to output ID3 file
 
+optional arguments:
+  -h, --help   show this help message and exit
 ```
```
Comment (ljdursi) [inline README.md:57]: This is good; could you maybe add:

A specific case: e.g.
```
train-id3 test-cases/case3/config.json --diagram case3.png
predict-id3 test-cases/case3/test-config.json 
```

Explain what it does (e.g. the different config files) and show the diagram that results?

**Review by ljdursi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Samarth-Agarwal1) [reply README.md]: Actually, now that you bring it up, there isn't really a reason. This is just the way I have been doing it whenever I want to build it.

Comment (ljdursi) [reply README.md]: It's an important step for distributing the wheel (e.g. through pypi) but for getting a user up and running quickly, the simpler step is probably easier.

Comment (Samarth-Agarwal1) [reply README.md]: So should I remove the Building section entirely and change the Installing section to pip install .? Or should I keep the Building section and just change the Installing section to pip install .?

Comment (ljdursi) [reply README.md]: Maybe merge them into either "Installing" or "Setup and Install" or something like that.  You could include creating a virtualenv, which is a good idea, and would expand the setup part

**Review by ljdursi — APPROVED** [review]
Great!  This looks terrific

**Pull request merged**